src/thread.rs

```diff
@@ -407,6 +446,18 @@ impl<'mir, 'tcx: 'mir> ThreadManager<'mir, 'tcx> {
         // We have not found a thread to execute.
         if self.threads.iter().all(|thread| thread.state == ThreadState::Terminated) {
             unreachable!();
+        } else if let Some(next_call_time) =
```

Comment:
The type of `next_call_time` is `TimeoutCallbackInfo`, right? If yes, please rename to `next_callback_info` or so.